cmd/util.go

```diff
@@ -12,10 +12,21 @@ import (
 	apiv1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+func newTrue() *bool {
+	b := true
+	return &b
+}
+
+func newFalse() *bool {
```

Comment:
using new(bool) would yield the same thing as this function

Reply:
oh nice, I will totally do that!